lcm-cmake/lcmUtilities.cmake

```diff
@@ -7,7 +7,9 @@
 #                  [JAVA_SOURCES <VARIABLE_NAME>]
 #                  [PYTHON_SOURCES <VARIABLE_NAME>]
 #                  [LUA_SOURCES <VARIABLE_NAME>]
+#                  [GO_SOURCES <VARIABLE_NAME>]
 #                  [DESTINATION <PATH>]
+#                  [GO_DESTINATION <PATH>]
 #                  <FILE> [<FILE>...])
```

Comment:
Due to the way Go uses `$GOPATH` (implicitly needs a `src`) we need to have the Go generated files inside a `src` sub directory.
This was the simplest way to achieve that.